gradle-sls-packaging/build.gradle

```diff
@@ -58,7 +58,7 @@ pluginBundle {
     }
 }
 
-def yourkitVersion = "2020.9-b411"
```

Comment:
yourkit deleted this. Should probably mirror this internally and avoid using the download task so we don't suffer from random remote deletions again (also Download task sucks).

Reply:
Scratch that, can't mirror this internally when this project is open source.